styles/main.css

```
@@ -41,6 +52,65 @@ button.primary {
+}
+
+.hidden {
+  display: none;
+}
+
+.pretty-search-bar {
+display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 480px; 
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)